### DISCUSSION TITLE
In scroll mode how to enter visual mode?

### DISCUSSION THREAD

**Discussion by semi710** [zellij-org/zellij#2385] [Q&A]
when i am in scroll mode how can i enable select or vim visual mode?
Currently the work around i use is to open the scrollback in  editormode then copying the text.

(and also is there a way to open available urls on terminal in browser from keyboard?)

**Comment by milanglacier** [comment]
I guess dump the buffer into the `$EDITOR` is the intended way of zellij to do "visual mode".

Instead of implementing a half-completed "vi copy mode", it is btter to use the real vim to manipulate the text of the buffer.
